### ISSUE TITLE
What is the goal for a 1.0 release?

### ISSUE THREAD

**Issue by cah-briangantzler** [miragejs/miragejs#1046]
The next steps Im doing for extracting pretender will be a breaking change. But since we are still a pre 1.0 release technically everything can be a breaking change.

Has there been any thought given as to what the goal is to release a 1.0?

**Comment by IanVS** [comment]
I haven't given it thought up to now.  My immediate response would be to release the next breaking change as 0.2.0, and let that bake for a little while, since it will be a fairly large change to mirage.  Then, once we're certain it's stable, and maybe we have some integrations for msw or other mocking layers, we could escape https://0ver.org/ and declare a 1.0, capitalizing on the event to share the news of the new flexibility that you're currently enabling.  

Or, do you have other thoughts / ideas?

**Comment by cah-briangantzler** [comment]
Integrations for msw or other mocking layers are already in place. No future updates needed. FYI

The upcoming changes are removing the code and requiring them to use an external library for mocking. But right now there is an mws one and you can use it at of `0.1.44`

**Comment by cah-briangantzler** [comment]
Agreed, the things you listed above will be in the next update I do that will be the breaking change, requiring the interceptor be specified
